### PR TITLE
Kids-254 Continue without VPC if not creating any children

### PR DESCRIPTION
**Pull request by Daniela510**
## Description

Create member without vpc if no children are being created.

### CODE REVIEW

File: lib/features/children/add_member/cubit/add_member_cubit.dart [hunk 1]
```diff
@@ -49,6 +49,16 @@ class AddMemberCubit extends Cubit<AddMemberState> {
   }
 
   void allFormsFilled() {
+    if (state.members.where((element) => element.type == 'Child').isEmpty) {
```
Comment (pavelpavelko) [inline lib/features/children/add_member/cubit/add_member_cubit.dart:52]: another good candidate for getters proposed in another ;)

Comment (pavelpavelko) [reply lib/features/children/add_member/cubit/add_member_cubit.dart:52]: also I'm a bit confused that now we have both Profile and Member models in the app.. Could you please provide more historical context here?

**Review by pavelpavelko — APPROVED** [review]
looks good with one small suggestion and also more general question (but it should not block merge oc)

File: lib/features/children/add_member/cubit/add_member_cubit.dart (hunk 1 above)
Comment (Daniela510) [reply lib/features/children/add_member/cubit/add_member_cubit.dart]: Member is the model we need to add a member to the family group: children or adults. Profile is what we get returned from BE including monster picture & wallet amount.

**Pull request merged**